### DISCUSSION TITLE
X.COM Login Screen Not Loading

### DISCUSSION THREAD

**Discussion by alfredogrind** [zen-browser/desktop#1512] [General]
Is there any reason that when i try to log into X.COM the main screen keeps Loading eternally?
![Screenshot from 2024-09-17 12-36-53](https://github.com/user-attachments/assets/b07cd689-24e2-47c8-9ac1-89eb40e4550f)

**Comment by hbekkouche** [comment]
same problem for me last version 1.0.1-a.2 (64-bit)

**Comment by hbekkouche** [comment]
i fixed it by disabling Firefox protection

![Screenshot 2024-10-07 113815](https://github.com/user-attachments/assets/02a927a7-3134-4249-bf8c-c416191cf895)
